src/App.js

```diff
@@ -36,16 +37,20 @@ function App(props) {
           exact
           component={localStorage.getItem("token") ? CreateVideo : Signup}
         />
+        <Route exact path="/users/:username" component={UserProfile} />
```

Comment:
If `:username` doesn't work, then maybe let's try `${username}`, or `localStorage.getItem("username")`. Currently the action doesn't support his when you login/register, so we'll need to add `localStorage.setItem("username", res.data.signIn.username);`